### PR TITLE
PORTALS-3088

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Screen Shot 2024-05-23 at 10 12 57 AM

Screen Shot 2024-05-23 at 11 10 37 AM

Screen Shot 2024-05-23 at 11 12 43 AM

Screen Shot 2024-05-23 at 11 14 45 AM

Screen Shot 2024-05-23 at 11 16 16 AM

Screen Shot 2024-05-23 at 11 18 06 AM

File: apps/SageAccountWeb/src/components/AccountSettings.tsx [hunk 1]
```diff
@@ -572,8 +585,30 @@ export const AccountSettings = () => {
                 </div>
                 <div className="credential-partition">
                   <h4>Profile Validation</h4>
-                  <CompletionStatus isComplete={verified} />
-                  <p>
+                  {(verificationState === undefined ||
+                    verificationState.state ===
+                      VerificationStateEnum.APPROVED) && (
+                    <CompletionStatus isComplete={verified} />
+                  )}
```
Comment (nickgros) [inline apps/SageAccountWeb/src/components/AccountSettings.tsx:592]: Is this appropriate to show if `verificationState === undefined`? I would expect a skeleton (or nothing)

**Review by nickgros — APPROVED** [review]

File: apps/SageAccountWeb/src/components/AccountSettings.tsx (hunk 1 above)
Comment (jay-hodgson) [reply apps/SageAccountWeb/src/components/AccountSettings.tsx:592]: This is the case before a submission has been created, so verified is false (and this UI would show Incomplete in red).

Comment (nickgros) [reply apps/SageAccountWeb/src/components/AccountSettings.tsx:592]: I completely missed the `isComplete` prop (which didn't even change). Totally makes sense!

**Pull request merged**